Build/Packages.json

```diff
@@ -0,0 +1,14 @@
+{
+    "AppTranslations": {
+        "Id": "Microsoft.Dynamics.BusinessCentral.Translations",
```

Comment:
`Name` instead of `Id`?

Reply:
I think Id is valid :) But why do we need AppTranslations/AppBaselines? Couldn't the structure not be: 
"Microsoft.Dynamics.BusinessCentral.Translations" : { "Version": "22.1.20230503.4" }

Reply:
It's mainly because the package name comes from outside and I'd prefer to have it in one spot in the code.

Reply:
Is it an issue that it comes from the outside?

Reply:
It's not an issue. It's mainly to have the name in a single place, in case it changes.
However, as we kind of own these packages anyway, I'll use the names around. 

See latest changes on the `packages.json`.